app/forms/draft_collection_form.rb

```diff
@@ -6,6 +6,11 @@ class DraftCollectionForm < Reform::Form
   extend T::Sig
   feature EmbargoDate
 
+  EMBARGO_RELEASE_DURATION_OPTIONS = [['6 months from date of deposit', '6 month'],
+                                      ['1 year from date of deposit', '1 year'],
+                                      ['2 years from date of deposit', '2 years'],
+                                      ['3 years from date of deposit', '3 years']].freeze
```

Comment:
I don't have strong feelings about this, but if you convert this to a hash (which `options_for_select` should be able to use just as well), you could change the `.map(&:last)` on line 47 to `.values`. Maybe clearer? Maybe not. What do you think?

Reply:
Yes, that's better.  I'll make it a hash.